src/environmentd/src/http.rs

```diff
@@ -673,16 +673,19 @@ async fn auth<B>(
 pub struct InternalServer {
     metrics_registry: MetricsRegistry,
     otel_enable_callback: OpenTelemetryEnableCallback,
+    stderr_filter_callback: StderrFilterCallback,
```

Comment:
Honestly, I would just make these types the same, and just do a breaking change to the api of `/api/opentelemtry/config`, its not stable anyways